sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/OpenAIAsyncClient.java

```diff
@@ -1286,9 +1286,8 @@ public Mono<Response<String>> getAudioTranslationTextWithResponse(String deploym
      * @return transcribed text and associated metadata from provided spoken audio data along with {@link Response} on
      * successful completion of {@link Mono}.
      */
-    @Generated
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<Response<BinaryData>> getAudioTranscriptionAsPlainTextWithResponse(String deploymentOrModelName,
+    Mono<Response<BinaryData>> getAudioTranscriptionAsPlainTextWithResponse(String deploymentOrModelName,
```

Comment:
Instead of manually remove the generated annotation, do you know why it was generated initially? It seems a possible to not regenerate this `AsPlainText` methods instead of manually remove the generated.

Reply:
This was due to the `MultipartFormDataRequestHeadersTraits` in typespec not having effect on `Azure.Core.Foundations.ResourceOperation` template.

I was attempted to address this in spec PR https://github.com/Azure/azure-rest-api-specs/pull/26412 but didn't get response from Travis (mostly to clarify whether the `content-type` header in response be `text/plain`).

Sure, let me try again if I can revive that spec PR.